backend/fms_core/migrations/0058_v4_5_0.py

```diff
@@ -0,0 +1,160 @@
+import reversion
+from django.contrib.auth.models import User
+from django.db import migrations
+from fms_core.models._constants import SampleType
+
+ADMIN_USERNAME = 'biobankadmin'
+
+def create_axiom_experiment_run_related_objects(apps, scheme_editor):
+    Platform = apps.get_model("fms_core", "Platform")
+    InstrumentType = apps.get_model("fms_core", "InstrumentType")
+    Instrument = apps.get_model("fms_core", "Instrument")
+    RunType = apps.get_model("fms_core", "RunType")
+    Protocol = apps.get_model("fms_core", "Protocol")
+    PropertyType = apps.get_model("fms_core", "PropertyType")
+    ContentType = apps.get_model('contenttypes', 'ContentType')
+    Step = apps.get_model("fms_core", "Step")
+    ProtocolBySubprotocol = apps.get_model("fms_core", "ProtocolBySubprotocol")
+    Workflow = apps.get_model("fms_core", "Workflow")
+    StepOrder = apps.get_model("fms_core", "StepOrder")
+
+    with reversion.create_revision(manage_manually=True):
+        admin_user = User.objects.get(username=ADMIN_USERNAME)
+        admin_user_id = admin_user.id
+        
+        # Create Platform and InstrumentType
+        platform = Platform.objects.create(name="Axiom", created_by_id=admin_user_id, updated_by_id=admin_user_id)
+        reversion.add_to_revision(platform)
+
+        # DICT { TYPE : PLATFORM }
+        INSTRUMENT_TYPES = {
+            "GeneTitan": "Axiom",
```

Comment:
replace "GeneTitan" by "GeneTitan MC". MC stands for Multi-Channel and might help differentiate with future instrument of this line.